backwardcompatibilityml/widget/resources/index.html

```diff
@@ -0,0 +1,37 @@
+<!--
```

Comment:
Is there a reason that we cannot re-use the existing widget.html file?

Reply:
widget.html is a Jinja template, which won't work with `webpack dev server` (unless there's some clever plugin that allows us to do that)

Reply:
Ah, I understand.

Ok, so since we need to run both the webpack dev server and the Python fixture generation code anyway. Could we do this instead:

1. Have the Python fixture generation code, output the HTML (generated from the Junja2 template) and place that in the folder.

2. Have the Webpack dev server run and serve things up from that location.

Reply:
I think we can do that. Are you anticipating needing to inject more things into the template later on? (I noticed "data" is always empty)

Reply:
Yeah the injection needs to be cleaned up. It is always possible. Part of the reason for suggesting this is so that we have as minimal differences as possible between how we load the widget in Jupyter vs the dev harness.